src/store/theme-cache.ts

```diff
@@ -0,0 +1,10 @@
+const key = 'CURRENT_THEME';
+
+export function saveTheme(theme: string) {
```

Comment:
There's an enum type for theme, perhaps use that here instead of the string type.